modules/sample.cpp

```diff
@@ -91,16 +91,15 @@ class CSampleMod : public CModule {
     }
 
     void OnIRCConnected() override {
-        PutModule(t_s("You got connected BoyOh."));
+        PutModule(t_s("You got connected."));
     }
 
     void OnIRCDisconnected() override {
-        PutModule(t_s("You got disconnected BoyOh."));
+        PutModule(t_s("You got disconnected."));
     }
 
     EModRet OnIRCRegistration(CString& sPass, CString& sNick, CString& sIdent,
                               CString& sRealName) override {
-        sRealName += " - ZNC";
```

Comment:
This is a sample module - to show what can be done in a module, and how approximately to do it. With `return CONTINUE` only, this callback does nothing, and should be removed. Maybe replace this line with something more useful if you don't like appending "-ZNC" to realname?